client/core/types.go

```diff
@@ -695,6 +695,18 @@ type OrderFilter struct {
 	Statuses []order.OrderStatus `json:"statuses"`
 }
 
+// Account holds data returned from AccountExport.
+type Account struct {
+	Host      string `json:"host"`
+	AccountID string `json:"accountID"`
+	PrivKey   string `json:"privKey"`
+	PubKey    string `json:"pubKey"`
```

Comment:
Let's rename `PubKey` to `DEXPubKey`, otherwise it looks like this is the pubkey that goes with the user's `PrivKey`, but it's the DEX server's public key.